packages/internet-header/cypress/percy/snapshot-configs/all.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
If we use this files in multiple packages, it would be nice to store them somewhere in the main repo, instead of save them in every package, where they may be used.